### PR TITLE
Admin update home page

### PR DESCRIPTION
**Pull request by sainikhilnaru**
This feature allows you to update the home page with the table insertions in the ADMIN tab.
This allows you to access the postgresql database from the html sections text and edit them from a tab.

### CODE REVIEW

File: vwb/app/controllers/edithomepages_controller.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
+class EdithomepagesController < ApplicationController
```
Comment (TauJiang) [inline vwb/app/controllers/edithomepages_controller.rb:1]: fulfill the acceptance criteria that the admin could change the home pages

File: vwb/app/views/edithomepages/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<%= link_to "Back to Home page", edithomepages_path %>
```
Comment (TauJiang) [inline vwb/app/views/edithomepages/edit.html.erb:1]: good basic views setup

File: vwb/app/views/edithomepages/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,22 @@
+
```
Comment (TauJiang) [inline vwb/app/views/edithomepages/index.html.erb:1]: has necessary CRUD functions integrated, meeting acceptance criteria

File: vwb/app/views/home/index.html.erb [hunk 4]
```diff
@@ -57,6 +57,17 @@
       <div class="row">
         <div class="col-lg-8 mx-auto">
           <h2 id="aboutH">About Us</h2>
+
```
Comment (TauJiang) [inline vwb/app/views/home/index.html.erb:60]: good changes

File: vwb/db/seeds.rb [hunk 5]
```diff
@@ -6,5 +6,14 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
+Edithomepage.create(name: 'home', description: 'This is the home description')
```
Comment (TauJiang) [inline vwb/db/seeds.rb:9]: initialize the database

**Review by TauJiang — APPROVED** [review]
Code reviewed by Yingtao Jiang.

File: vwb/app/controllers/edithomepages_controller.rb (hunk 1 above)
Comment (sainikhilnaru) [reply vwb/app/controllers/edithomepages_controller.rb:1]: Will add that criteria and push changes

**Pull request merged**